Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1516,6 +1516,249 @@ def broadcast_message():
+        context,
+        affected_agents
+    )
+
+
+def uninstall_agent_request(query=None, agent_id=None, group_id=None):
+
+    endpoint_url = 'agents/actions/uninstall'
+    filters = {}
+
+    if query:
+        filters['query'] = query
+    if agent_id:
+        filters['ids'] = agent_id
+    if group_id:
+        filters['groupIds'] = group_id
+
+    payload = {
+        "filter": filters,
+        "data": {}
+    }
+
+    response = http_request('POST', endpoint_url, data=json.dumps(payload))
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    else:
+        return response
+
+
+def uninstall_agent():
+    """
+    Sends an uninstall command to all agents matching the input filter.
+    """
+    query = demisto.args().get('query', '')
+
+    agent_id = argToList(demisto.args().get('agent_id'))
+    group_id = argToList(demisto.args().get('group_id'))
+
+    affected_agents = shutdown_agents_request(query, agent_id, group_id)
+    agents = affected_agents.get('data', {}).get('affected', 0)
+    if agents > 0:
+        contents = {
+            'ID': agent_id
+        }
+    else:
+        return_error('No agents were affected.')
+
+    context = {
+        'SentinelOne.Agent(val.ID && val.ID === obj.ID)': contents
+    }
+
+    return_outputs(
+        f' Uninstall was sent to {agents} agent(s).',
+        context,
+        affected_agents
+    )
+
+
+# Event Commands
+
+def create_query_request(query, from_date, to_date):
+
+    endpoint_url = 'dv/init-query'
+    payload = {
+        "query": query,
+        "fromDate": from_date,
+        "toDate": to_date
+    }
+
+    response = http_request('POST', endpoint_url, data=json.dumps(payload))
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    else:
+        return response.get('data').get('queryId')
+
+
+def create_query():
+
+    query = demisto.args().get('query')
+    from_date = demisto.args().get('from_date')
+    to_date = demisto.args().get('to_date')
+
+    query_id = create_query_request(query, from_date, to_date)
+
+    demisto.results('The query ID is ' + str(query_id))
+
+
+def get_events_request(query_id=None, limit=None):
+
+    endpoint_url = 'dv/events'
+
+    params = {
+        'query_id': query_id,
+        'limit': limit
+    }
+
+    response = http_request('GET', endpoint_url, params)
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    if 'data' in response:
+        return response.get('data')
+    return {}
+
+
+def get_events():
+    """
+    Get all Deep Visibility events from query
+    """
+    contents = []
+    headers = ['EventType', 'AgentName', 'SiteName', 'User', 'Time', 'AgentOS', 'ProcessID', 'ProcessUID',
+               'ProcessName', 'MD5', 'SHA256']
+    query_id = demisto.args().get('query_id')
+    limit = int(demisto.args().get('limit', 50))
+
+    events = get_events_request(query_id, limit)
+    if events:
```

Comment:
need to handle the `else`, like if there are no events, no need to return a table